src/js/ongoingMovies.js

```diff
@@ -0,0 +1,112 @@
+import '../styles/ongoingMovies.scss';
+import { initializeMovieData, moviesArray } from './main.js';
+
+const ongoingMoviesDom = document.querySelector(".ongoingMovies"); 
+
+let genres = [] 
+
+let testArray = [
+    {
+        src: "https://media.istockphoto.com/id/1147544807/sv/vektor/ingen-miniatyr-bild-vektor-grafik.jpg?s=612x612&w=0&k=20&c=RWp5ECPYkIEO8J3zxyvsc4Lrf68NiLDybtPHix_QI1I=",
+        movieLabel: "Sune i fjällen"
+    },
+    {
+        src: "https://media.istockphoto.com/id/1147544807/sv/vektor/ingen-miniatyr-bild-vektor-grafik.jpg?s=612x612&w=0&k=20&c=RWp5ECPYkIEO8J3zxyvsc4Lrf68NiLDybtPHix_QI1I=",
+        movieLabel: "Sune i Norge"
+    },
+    {
+        src: "https://media.istockphoto.com/id/1147544807/sv/vektor/ingen-miniatyr-bild-vektor-grafik.jpg?s=612x612&w=0&k=20&c=RWp5ECPYkIEO8J3zxyvsc4Lrf68NiLDybtPHix_QI1I=",
+        movieLabel: "Sune i fjällen"
+    },
+    {
+        src: "https://media.istockphoto.com/id/1147544807/sv/vektor/ingen-miniatyr-bild-vektor-grafik.jpg?s=612x612&w=0&k=20&c=RWp5ECPYkIEO8J3zxyvsc4Lrf68NiLDybtPHix_QI1I=",
+        movieLabel: "Sune i Norge"
+    },
+]
+
+const useData = async () => {
+    await initializeMovieData(); // Wait for the data to load
+};
+
+useData();
+
+setTimeout(() => {
+    InitializeOngoingMovies();
+}, 500);
+
+function InitializeOngoingMovies() {
+    createGenres();
+
+    createFilterProps();
+
+    for (let currentIndex = 0; currentIndex < testArray.length; currentIndex++) {
+        const element = testArray[currentIndex];
+        createMovieCard({
+            src: element.src,
+            movieLabel: element.movieLabel
+        })
+    };
+}
+
+function createMovieCard(props) {
+    let cardDiv = document.createElement("div");
```

Comment:
Impressive work with finding appropriate genres!
You could change the variable type from 'let' to 'const', when creating elements such as; 'document.createElement("div");' etc. Though this is not necessary.

The API isn't always loading when entering the page, it is a weird bug affecting all API retrievals. I do not know the cause of this bug, but it I **don't** think your code is the cause of this...

Otherwise great job overall Magnus! :)

Reply:
![Screenshot 2024-12-30 at 22 48 14](https://github.com/user-attachments/assets/0f28c30f-a188-4542-a02a-4718d16d5a4f)

Reply:
### Mobile view
![Screenshot 2024-12-30 at 23 42 33](https://github.com/user-attachments/assets/a2d4fa72-94f9-4952-a86e-511ff3421483)

### Desktop view
![Screenshot 2024-12-30 at 23 42 25](https://github.com/user-attachments/assets/15f7f3b0-3da0-432f-a0df-37fa6c75b05a)

Reply:
Great suggestion with changing the variable types! I've now changed the variables types from let to const in these cases 💯 

Looking for a fix on the API not loading in before initialization. Thank you!

Reply:
I believe i found a fix for the API not loading using DomContentLoaded. Tried refreshing the server & website multiple times without getting this problem again

Reply:
It looks as if the bug is fixed! Very good job Magnus! Happy New Year! :)